plugins/main/public/components/endpoints-summary/services/get-agents-info.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  UI_LOGGER_LEVELS,
+  UI_ORDER_AGENT_STATUS,
+} from '../../../../common/constants';
+import { getErrorOrchestrator } from '../../../react-services/common-services';
+import { UI_ERROR_SEVERITIES } from '../../../react-services/error-orchestrator/types';
+import { WzRequest } from '../../../react-services/wz-request';
+import { getColorPaletteByIndex } from './get-color-palette-by-index';
+import {
+  agentStatusLabelByAgentStatus,
+  agentStatusColorByAgentStatus,
+} from '../../../../common/services/wz_agent_status';
+
+export const getAgentsInfo = async () => {
+  const DEFAULT_COUNT = 1;
+  try {
+    // Uncomment before merge
+    // const { data }: any = await WzRequest.apiReq('GET', '/decoders', '{}');
```

Comment:
Be mindful that the response of the API has the following structure:
```json
{
  "data": {
    "status": {
      "active": 8,
      "disconnected": 0,
      "never_connected": 1,
      "pending": 2
    },
    "os": {
      "debian": 8,
      "ubuntu": 3
    },
    "groups": {
      "default": 11,
      "test": 5
    }
  },
  "error": 0
}
```

Therefore the response `data` object has a nested property called `data` as well.